eng/Build.props

```diff
@@ -140,6 +140,7 @@
                           $(RepoRoot)src\Servers\**\*.csproj;
                           $(RepoRoot)src\Security\**\*.*proj;
                           $(RepoRoot)src\SiteExtensions\Microsoft.Web.Xdt.Extensions\**\*.csproj;
+                          $(RepoRoot)src\SiteExtensions\LoggingAggregate\test\**\*.csproj;
```

Comment:
Was this test previously skipped because the project wasn't built?

Reply:
I think it was just missed? I'm not sure